client/pyproject.toml

```
@@ -7,6 +7,9 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "3.10.*"
+Kivy = "^2.1.0"
```

sorry for the dependencies, we need to change tool poetry name
```suggestion
[tool.poetry]
name = "Blak"
version = "0.1.0"
description = "itsn't a bug its a feature"
authors = ["Author 1 <Gorgeous-Ghouls@github.com>"]
license = "MIT"
```

---

I don't understand it

---

Ok i get it, will do